city_scrapers/spiders/chi_ohare_noise.py

```diff
@@ -0,0 +1,212 @@
+                source=response.url,
+            )
+
+            meeting = self._get_status(meeting)
+            meeting = self._parse_classification(meeting)
+            return meeting
+
+        def _parse_title(self, response):
+            """Parse or generate meeting title."""
+            return response.xpath(
+                "//div[@class='jev_evdt_header']/div/h2/text()"
+            ).extract()[0]
+
+        def _parse_url(self, response):
+            """Parse or generate meeting title."""
+            return [i.strip() for i in response.xpath("p/a/@href").extract()][0]
+
+        def _parse_description(self, response):
+            """Parse or generate meeting description."""
+            return response.xpath("//div[@class='jev_evdt_desc']/p/text()").get() or ""
+
+        def _parse_classification(self, meeting):
+            """Parse or generate classification from allowed options."""
+            if "committee" in meeting["title"].lower():
+                meeting["classification"] = COMMITTEE
+            elif "commission" in meeting["title"].lower():
+                meeting["classification"] = COMMISSION
+            else:
+                meeting["classification"] = NOT_CLASSIFIED
+            return meeting
+
+        def _parse_start(self, response):
+            """Parse start datetime as a naive datetime object."""
+            return datetime.strptime(
+                " ".join(
+                    [
+                        i.strip()
+                        for i in response.xpath(
+                            "//div[@class='jev_evdt_header']/div/p/text()"
+                        ).extract()
+                    ]
+                ),
+                "%A, %B %d, %Y %I:%M%p",
+            )
+
+        def _parse_all_day(self, response):
+            """Parse or generate all-day status. Defaults to False."""
+            return False
+
+        def _parse_location(self, response):
+            """Parse or generate location."""
+            addr = re.split(
+                "-|,",
+                response.xpath("//div[@class='jev_evdt_location']/text()")
+                .extract()[0]
+                .strip(),
+                maxsplit=1,
+            )
+            return {
+                # Using reverse indexing for the cases
+                # where there is no building name or no location
+                "address": addr[-1].strip(),
+                "name": addr[0].strip(),
+            }
+
+        def _parse_links(self, response):
+            """Parse or generate links."""
+            return [
+                {"href": item, "title": "Zoom Link"}
+                for item in response.xpath(
+                    "//div[@class='jev_evdt_desc']/p/a/@href"
+                ).extract()[:1]
+            ]
+
+        def _parse_source(self, response):
+            """Parse or generate source."""
+            return response.url
+
+        def _get_status(self, meeting):
```

Comment:
Missed this when looking at the implementation above, but this functionality is all contained in the `CityScrapersSpider` `get_status` method if you pass the title text to check for cancellation info to the `text` kwarg